doc/building_applications.rst

```diff
@@ -107,11 +104,19 @@ Trivial Example: `enclave.c`
             printf("beta started\n");
     }
 
-Configuring Cmake
+Configuring CMake
 -----------------
 
-To build a project with PIRATE LLVM specify the compile with `CC` and `CXX` environment
-variables as seen above.
+To build a project with PIRATE LLVM set the ``CC`` and ``CXX`` to the PIRATE LLVM toolchain.
+For example,
+
+.. code-block:: sh
+
+  CC=`realpath llvm-build/bin/clang`
+  CXX=`realpath llvm-build/bin/clang++`
```

Comment:
Should this include `export`?

Reply:
I think the preferred method is to prepend CC=<clang> CXX=<clang++> to the compile command so that the variables are only set for this compile. For example

` CC=<clang> CXX=<clang++> make`

I can include an example that shows this?

Reply:
Yeah, I think that would be a good idea.

Reply:
I decided to just leave the environment variables out of it. The generator-agnostic CMake way to do it is to define `CMAKE_C_COMPILER` and `CMAKE_CXX_COMPILER` so I just gave an example of that.